packages/firestore/src/lite-api/expressions.ts

```diff
@@ -5867,10 +5868,10 @@ export function strConcat(
  * ```
  *
  * @param fieldName The field name of the map field.
- * @param subField The key to access in the map.
+ * @param key The key to access in the map.
```

Comment:
subField doesn't seem quite right within the context of maps. I kinda see where you are going, in which case maybe subFieldName might be more accurate within the context of documents. I personally prefer the word key since it has good programatic meaning.